textproc/lowdown/Portfile

```diff
@@ -1,26 +1,22 @@
-PortSystem		1.0
-PortGroup		snowleopard_fixes 1.0
+PortSystem              1.0
 
-name			lowdown
-categories		textproc
-version			0.3.1
-revision		1
-license			BSD
+name                    lowdown
+categories              textproc
+version                 0.3.2
+license                 BSD
 
-description		simple markdown translator
-long_description	lowdown translates markdown to either HTML	\
-			or a roff document using either -man or -ms
-homepage		http://kristaps.bsd.lv/lowdown/
-master_sites		http://kristaps.bsd.lv/lowdown/snapshots/
-maintainers		{stare.cz:hans @janstary}
-platforms		openbsd darwin linux
+description             simple markdown translator
+long_description        lowdown translates markdown to either HTML	\
```

Comment:
FYI, there's still a tab at the end of that line...

Reply:
(Thanks for cleaning this up!)

Reply:
eye-ripping tab removed, squashed.